# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../flex/gen.c-fix.patch
# Copyright (C) 2006 The OpenSDE Project
# Copyright (C) 2004 - 2006 The T2 SDE Project
# Copyright (C) 1998 - 2004 Clifford Wolf
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- flex-2.5.33/gen.c.orig	2006-08-29 23:50:40.000000000 -0600
+++ flex-2.5.33/gen.c	2006-08-30 00:37:53.000000000 -0600
@@ -1810,11 +1810,9 @@
 	if (yymore_used) {
 		if (!C_plus_plus) {
 			if (yytext_is_array) {
-				if (!reentrant){
-    				indent_puts ("static int yy_more_offset = 0;");
-                    indent_puts ("static int yy_prev_more_offset = 0;");
-                }
-			}
+				indent_puts ("static int yy_more_offset = 0;");
+				indent_puts ("static int yy_prev_more_offset = 0;");
+			}		
 			else if (!reentrant) {
 				indent_puts
 					("static int yy_more_flag = 0;");
